### DISCUSSION TITLE
6.26 일일 회의

### DISCUSSION THREAD

**Discussion by t4e1** [3sam5oh/openSearch-mini-project#4] [일일 회의]
## 1. 주제 선정
화면 레이아웃 관리를 위한 세션 서비스 기획
웹툰 쪽 관련된 분이 요구 사항을 적은 게 아닐지? 
 - 네이버 웹툰 생각해보면 남성/ 여성 / 연령대 별 인기 순을 적용할 수 있지 않을까? 
 - 무신사(나이 대 별 추천) 

### 섹션 플랫폼 
어떤 주제를 선정할 것인지
1. 웹툰
2. 이커머스  

#### 이커머스 
1. 카페 메뉴 검색  (선정)
2. 개발 교육 검색 
3. 주류 검색 

#### 서비스 기획 (추후 대용량 시스템 처리 프로젝트 - 이벤트 기반 가차 시스템)
1. 웹 구현 but 모바일 화면을 중점적으로 고려
2. 위치 기반 서비스
3. 검색바
4. 메뉴 선택에 따른 랭킹 시스템
      - 리뷰 기반
      - 좋아요 기반
      - 검색 횟수 기반
      - 조회수 기반
      - 찜하기
      비즈니스 로직: 메뉴 검색 (ex, 아메리카노) 을 했을 때 사용자 위치 (최우선 기반) 를 기반으로 주변 카페를 보여줌, 태그에 따른 상위 추천
5. 메뉴 추천
      - 사용자 커스터마이즈 추천
      - 카테고리 추천
      - gpt 추천
      
6. api 참고 링크
- https://www.coocon.net/main_0001_t5.act
- https://www.data.go.kr/index.do

**Comment by jlee38266** [comment]
카페 메뉴 좋은 것 같습니다!

**Comment by t4e1** [comment]
세미 프로젝트 : 5번 위주
